tests/baselines/reference/jsxFragmentFactoryReference(jsx=react-jsx).errors.txt

```diff
@@ -0,0 +1,15 @@
+jsxFragmentFactoryReference.tsx(3,9): error TS2792: Cannot find module 'react/jsx-runtime'. Did you mean to set the 'moduleResolution' option to 'nodenext', or to add aliases to the 'paths' option?
+jsxFragmentFactoryReference.tsx(3,9): error TS2879: Using JSX fragments requires fragment factory 'React' to be in scope, but it could not be found.
+
+
+==== jsxFragmentFactoryReference.tsx (2 errors) ====
+    export class LoggedOut {
+        content = () => (
+            <>        </>
+            ~~
+!!! error TS2792: Cannot find module 'react/jsx-runtime'. Did you mean to set the 'moduleResolution' option to 'nodenext', or to add aliases to the 'paths' option?
+            ~~
+!!! error TS2879: Using JSX fragments requires fragment factory 'React' to be in scope, but it could not be found.

```

Comment:
Isn't this just... not true? Under `react-jsx`, `Fragment` is imported from the jsx runtime module (automatically) - `React` shouldn't be needed in scope at all, afaik.